lib/bindings/langs/react-native/src/gen_kotlin/templates/Helpers.kt

```diff
@@ -63,6 +63,21 @@ fun readableArrayOf(values: Iterable<*>?): ReadableArray {
     return array
 }
 
+fun asArrayList(arr: ReadableArray): ArrayList<Any> {
+    val arrayList = ArrayList<Any>();
+    for (i in 0..arr.size()-1) {
+        when (arr.getType(i)) {
```

Comment:
Seems like ReadableArray also has type Null. Should we assign null in this case? https://github.com/jackuhan/AndroidMonitor/blob/master/app/src/main/java/com/facebook/react/bridge/ReadableType.java#L18

Reply:
Even though we don't have a case were we have optional elements in an array, we should support at least the conversion to a List with nullable elements. Done [92ff98f](https://github.com/breez/breez-sdk-liquid/pull/482/commits/92ff98ff7889ff4de501cfbf2b146021d40e0027)